chef_master/source/release_notes_chefdk.rst

```diff
@@ -951,7 +943,6 @@ What's New in 1.0
 Version 1.0!
 -----------------------------------------------------
 
-We're recognizing ChefDK's continued stability with the honor of a 1.0 tag. There
```

Comment:
We probably want this

Reply:
```suggestion
We're recognizing ChefDK's continued stability with the honor of a 1.0 tag. There
```

Reply:
I might need you to hit the "commit suggestion" button, because it isn't showing for me.

Reply:
oh yeah no DCO this is a perfect place for it

Reply:
Nope you can't add back a line with suggestions. There's a few deleted lines that look like accidents

Reply:
Sadly, I don't think we can use it on deleted lines.